src/woo_publications/publications/tests/factories.py

```diff
@@ -12,6 +14,8 @@
 class PublicationFactory(factory.django.DjangoModelFactory[Publication]):
     publisher = factory.SubFactory(OrganisationFactory, is_actief=True)
     officiele_titel = factory.Faker("word")
+    bron_bewaartermijn = factory.Faker("word")
```

Comment:
I'd leave this empty in the factory - usually in factory you only specify things that must be set to a non-empty value. If you always populate a value, it creates the wrong impression in tests that this field cannot be empty, but t can.